receiver/kafkametricsreceiver/topic_scraper.go

```diff
@@ -82,35 +83,37 @@ func (s *topicScraper) scrape(context.Context) (pdata.Metrics, error) {
 		}
 		labels := pdata.NewAttributeMap()
```

Comment:
This variable can be removed now?

Reply:
Yes, this is unused. Removed it from consumer_scraper.go as well.
Thanks for catching this.